models/multiphase/d2q9_pf_velocity/conf.mk

```diff
@@ -0,0 +1,3 @@
+ADJOINT=0
+TEST=FALSE
+OPT="A+B"
```

Comment:
It would be good to have more meaningful names for the options? and a # comment on what is what in this file.